src/DocScan/Session/Create/Check/RequestedCustomAccountWatchlistAdvancedCaConfig.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\DocScan\Session\Create\Check;
+
+use Yoti\DocScan\Session\Create\Check\Contracts\Advanced\RequestedCaMatchingStrategy;
+use Yoti\DocScan\Session\Create\Check\Contracts\Advanced\RequestedCaSources;
+use Yoti\DocScan\Session\Create\Check\Contracts\RequestedWatchlistAdvancedCaConfig;
+
+class RequestedCustomAccountWatchlistAdvancedCaConfig extends RequestedWatchlistAdvancedCaConfig
+{
+    /**
+     * @var string
+     */
+    private $apiKey;
+
+    /**
+     * @var bool
+     */
+    private $monitoring;
+
+    /**
+     * @var string[]
+     */
+    private $tags;
+
+    /**
+     * @var string
+     */
+    private $clientRef;
+
+    /**
+     * RequestedCustomAccountWatchlistAdvancedCaConfig constructor.
+     * @param bool $removeDeceased
+     * @param bool $shareUrl
+     * @param RequestedCaSources $sources
+     * @param RequestedCaMatchingStrategy $matchingStrategy
+     * @param string $apiKey
+     * @param bool $monitoring
+     * @param string[] $tags
```

Comment:
@LuborRod A question for you, I have been basing the Node SDK mostly on your work, and I was wondering about this `tags`, as the documentation says it should be a map and it looks like you have a list of string (but I am not aware of Php, maybe there is something I am missing).
Documentation:

```"tags": { "someKey": "someValue" }, // optional map, see below```
```
Tags
The map of tags is dependent on the api_key - we have no way of knowing what should or should not be allowed.  We will only accept tags if they are giving us their api_key.  The map is optional, we will allow an empty map, but we will not allow…
null map
null keys or values
```

Reply:
@laurent-yoti In PHP, arrays serve as all types of containers. They can be used as maps, lists and sets.

Reply:
👍